src/coreclr/hosts/inc/coreclrhost.h

```diff
@@ -16,11 +16,17 @@
 
 #include <stdint.h>
 
+#ifdef __cplusplus
+#define CORECLR_HOSTING_API_LINKAGE extern "C"
+#else
+#define CORECLR_HOSTING_API_LINKAGE
+#endif
```

Comment:
FYI @elinor-fung

Reply:
Any action points for the follow-up work?

Reply:
I believe @AaronRobinsonMSFT has added this comment just for @elinor-fung's  awareness. There is nothing to change here. It just fixes a problem when this header is included from C code, because the `extern "C"` is C++ construct to make the function name mangling the same as in C.